spec/system/candidate_interface/candidate_apply_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe 'A candidate applying from Find' do
+  let(:provider_code) { '1AB' }
+  let(:course_code) { '2ABC' }
+  before do
+    visit "/candidate/apply" \
+      "?providerCode=#{provider_code}&courseCode=#{course_code}"
+  end
+
+  it 'sees the apply page' do
+    expect(page).to have_content t('apply.heading')
+  end
+
+  it 'sees their provider code' do
+    expect(page).to have_content provider_code
+  end
+
+  it 'sees their course code' do
+    expect(page).to have_content course_code
+  end
+
+  it 'can apply through UCAS' do
+    expect(page).to have_content t('apply.apply_button')
+  end
```

Comment:
we could simplify this group of tests using `subject`?
for example:

```
subject {page}

it { page.should have_content t('apply.heading') }
it { page.should have_content provider_code }
...
```

Reply:
I'm actually not a fan of that syntax as it's a bit harder to throw a `binding.pry` into it, I usually avoid using it. I don't think we should prescribe one or the other though and am happy to mix it in the codebase